Pysnirf2 has changed names to snirf

https://github.com/BUNPC/pysnirf2

```text
E   DeprecationWarning: Installation of this library via the pysnirf2 remote is deprecated after v0.7.3. Install future versions using `pip install snirf`.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@ ARG mne_nirs_v
 
 USER root
 
-RUN pip install statsmodels lets-plot dabest fooof h5io
+RUN pip install statsmodels dabest fooof h5io
 RUN pip install https://github.com/nilearn/nilearn/archive/${nilearn_v}.zip
 RUN pip install https://github.com/mne-tools/mne-bids/archive/${mne_bids_v}.zip
 RUN pip install https://github.com/mne-tools/mne-nirs/archive/${mne_nirs_v}.zip

--- a/examples/general/plot_19_snirf.py
+++ b/examples/general/plot_19_snirf.py
@@ -32,7 +32,7 @@ You can read the SNIRF protocol at the official site https://github.com/fNIRS/sn
 
 import os
 import mne
-import pysnirf2
+import snirf
 
 from mne.io import read_raw_nirx, read_raw_snirf
 from mne_nirs.io import write_raw_snirf
@@ -87,10 +87,10 @@ snirf_intensity.plot(n_channels=30, duration=300, show_scrollbars=False)
 #
 # To validate that a file complies with the SNIRF standard you should use the
 # official SNIRF validator from the Boston University Neurophotonics Center
-# called ``pysnirf2``. Detailed instructions for this program can be found at
+# called ``snirf``. Detailed instructions for this program can be found at
 # https://github.com/BUNPC/pysnirf2. Below we demonstrate that the files created
 # by MNE-NIRS are compliant with the specification.
 
-result = pysnirf2.validateSnirf('test_raw.snirf')
+result = snirf.validateSnirf('test_raw.snirf')
 assert result.is_valid()
 result.display()

--- a/mne_nirs/io/snirf/tests/test_snirf.py
+++ b/mne_nirs/io/snirf/tests/test_snirf.py
@@ -7,7 +7,7 @@ import datetime
 import h5py
 from numpy.testing import assert_allclose
 import pytest
-from pysnirf2 import validateSnirf
+from snirf import validateSnirf
 
 from mne.datasets.testing import data_path, requires_testing_data
 from mne.utils import requires_h5py, object_diff
@@ -64,6 +64,7 @@ def test_snirf_write(fname, tmpdir):
         if ('logno' not in line) and \
                 ('scanno' not in line) and \
                 ('his_id' not in line) and\
+                ('dig' not in line) and\
                 ('datetime mismatch' not in line):
             # logno and scanno are not used in processing
             diffs += f'\n{line}'

--- a/mne_nirs/statistics/tests/test_glm_type.py
+++ b/mne_nirs/statistics/tests/test_glm_type.py
@@ -20,7 +20,7 @@ from mne_nirs.experimental_design import make_first_level_design_matrix
 from mne_nirs.statistics import run_glm
 
 data_path = testing.data_path(download=False)
-subjects_dir = data_path + '/subjects'
+subjects_dir = data_path / '/subjects'
 
 
 def _get_minimal_haemo_data(tmin=0, tmax=60):

--- a/mne_nirs/visualisation/tests/test_visualisation.py
+++ b/mne_nirs/visualisation/tests/test_visualisation.py
@@ -21,8 +21,8 @@ from mne_nirs.utils import glm_to_tidy
 
 
 testing_path = testing.data_path(download=False)
-raw_path = testing_path + '/NIRx/nirscout/nirx_15_2_recording_w_short'
-subjects_dir = testing_path + '/subjects'
+raw_path = testing_path / '/NIRx/nirscout/nirx_15_2_recording_w_short'
+subjects_dir = testing_path / '/subjects'
 
 requires_mne_1 = pytest.mark.skipif(not check_version('mne', '1.0'),
                                     reason='Needs MNE-Python 1.0')

--- a/requirements_doc.txt
+++ b/requirements_doc.txt
@@ -19,7 +19,7 @@ Ipython
 sphinx-fontawesome
 seaborn
 fooof
-pysnirf2>=0.4.2
+snirf
 statsmodels
 dabest
 mne-qt-browser

--- a/requirements_testing.txt
+++ b/requirements_testing.txt
@@ -17,4 +17,4 @@ wheel
 mne-bids
 seaborn
 fooof
-pysnirf2>=0.3.1
+snirf

